addons/test/test-misra.py

```diff
@@ -120,6 +120,11 @@ def test_rules_suppression(checker, capsys):
         assert found is None, 'Unexptected output:\n' + captured
         dump_remove(src)
 
+def test_crash_misra9_parseInitializer():
```

Comment:
do you really need this test code?

It seems to me that cppcheck fails when running the command here:
https://github.com/danmar/cppcheck/blob/main/.github/workflows/CI-unixish.yml#L395

Reply:
I assume you didn't see the command and that this is redundant..